src/plugin/class-meta-fields-manager.php

```diff
@@ -35,11 +37,11 @@ public function register_meta_fields(): void {
 	/**
 	 * This function moves some of the meta fields to the post object before saving
 	 *
-	 * @param object          $prepared_post Post object that is going to be saved.
-	 * @param WP_REST_Request $request REST API request object.
+	 * @param object           $prepared_post Post object that is going to be saved.
+	 * @param \WP_REST_Request $request REST API request object.
 	 * @return object
 	 */
-	public function move_meta_fields( object $prepared_post, WP_REST_Request $request ): object {
+	public function move_meta_fields( object $prepared_post, \WP_REST_Request $request ): object {
```

Comment:
We could add `use` calls instead of namespacing when referencing.

Reply:
Vast majority of WordPress core code is in global namespace, so we can't use `use` here.

Reply:
I think I didn't explain correctly what I meant. Even if it's the global namespace, it's possible to do:

```php
namespace DotOrg\TryWordPress;

use WP_REST_Request;
```

Then in the rest of the file, whenever we need to reference `WP_REST_Request` we can omit the leading `\`.

Just something that I think makes the code easier to read, and also has the added advantage that you can tell what classes a file depends on by looking at all the `use` calls. 

Just a suggestion :)

Reply:
Ah, that's cool. Didn't realise we can do that. Will do!